docker/docker-compose.yml

```diff
@@ -17,14 +17,16 @@ services:
           "CMD",
           "node",
           "-e",
-          "require('http').get('http://localhost:3000/api/profile', (r) => {if (r.statusCode !== 200) throw new Error(r.statusCode)})"
+          "fetch('http://studio:3000/api/profile').then((r) => {if (r.status !== 200) throw new Error(r.status)})"
         ]
-      timeout: 5s
+      timeout: 10s
       interval: 5s
       retries: 3
     depends_on:
       analytics:
         condition: service_healthy
+    ports:
+      - "3000:3000"
```

Comment:
About the way we expose the ports in this file, on ubuntu, docker adds an iptables rule automatically that will expose 3000 on the network. Same for the other ports in this docker-compose.yml file. That's something to be aware of if you don't have an additional security group in front of your server.
On my own deployment I rewrite all the ports section to add "127.0.0.1:" in front of each port, for example "127.0.0.1:4000:4000" to not expose it on the network interface.
There is probably the same issue on the cli, we may just expose the different ports to any public Wi-Fi.

Reply:
Good catch this new export wasn't intended.

We might want to review the rest of the port expose in a dedicated PR, feel free to open an issue to discuss about this.